torchci/lib/github.ts

```diff
@@ -8,25 +7,11 @@ export async function getOctokit(
   owner: string,
   repo: string
 ): Promise<Octokit> {
-  let privateKey = process.env.PRIVATE_KEY as string;
+  let privateKey = process.env.PYTORCH_HUD_TOKEN as string;
   privateKey = Buffer.from(privateKey, "base64").toString();
 
-  const app = new App({
-    appId: process.env.APP_ID!,
-    privateKey,
-  });
-  const installation = await app.octokit.request(
-    "GET /repos/{owner}/{repo}/installation",
-    { owner, repo }
-  );
-
   return new Octokit({
-    authStrategy: createAppAuth,
-    auth: {
-      appId: process.env.APP_ID,
-      privateKey,
-      installationId: installation.data.id,
-    },
```

Comment:
Why is this constructor no longer needed?

Reply:
octokit has a bunch of authentication methods, this constructor is needed when using a github app but isnt needed when you just use a pat

https://octokit.github.io/rest.js/v18#authentication